package.json

```diff
@@ -40,7 +40,11 @@
     "create-modules:native": "node ./scripts/release/createNativeModules.js",
     "create-modules:web": "node ./scripts/release/createWebModules.js",
     "version": "ts-node --project ./scripts/tsconfig.json ./scripts/release/BumpVersion.ts",
-    "validate-staged-widget-versions": "node scripts/validation/validate-versions-staged-files.js"
+    "validate-staged-widget-versions": "node scripts/validation/validate-versions-staged-files.js",
+    "commit-msg": "commitlint --edit \"$1\"",
```

Comment:
Why is this dep added? It was recently removed https://github.com/mendix/widgets-resources/pull/1122

I think husky takes care of things here

Reply:
Don't know why this happened. Fixed.